### ISSUE TITLE
Add angular http call snippet in Generate Code Snippets

### ISSUE THREAD

**Issue by Ijaz-Tufail** [postmanlabs/postman-code-generators#345]
Hi, Please can u add an Angular HTTP call snippet in Generate Code Snippets, That will be very helpfull

**Comment by Abhirup-99** [comment]
I can work on this.

**Comment by Abhirup-99** [comment]
@umeshp7 can I work on this?

**Comment by Ijaz-Tufail** [comment]
Yes please

**Comment by Abhirup-99** [comment]
I wanted to ask how to do I compile it down to js. It would require angular core and the package has to be imported in appmodule, that is the case in the framework. This can be done syntatically but it can't be tested. Further thoughts would be helpful.
@umeshp7 @Ijaz-Tufail

**Comment by Ijaz-Tufail** [comment]
You can do this just like JQuery/AJAX call syntax. You can write code to
import the HttpClient module from "@angular/common/http" and write an Http
call syntax angular has provided.

**Comment by Abhirup-99** [comment]
The writing is not the problem, I am versed in angular. How to compile and run it, providing the syntax is not the issue. Adding tests and making sure it runs is the issue. @Ijaz-Tufail

**Comment by sheikhimtiaz** [comment]
@Abhirup-99 Hello, are you working on this issue? were you able to compile and run it?

**Comment by Abhirup-99** [comment]
> @Abhirup-99 Hello, are you working on this issue? were you able to compile and run it?

I couldn't compile it down. I can write the syntax down, but compiling it would require angular cli and the other stuff, which would be difficult to bring about.

**Comment by Ijaz-Tufail** [comment]
No, I'm not working on it. You have to create an angular app to test it.

**Comment by sheikhimtiaz** [comment]
Ok. I am working on it.

**Comment by ancgate** [comment]
Any updates on this feature?

**Comment by santidesimone** [comment]
Any updates on this feature?

**Comment by thiagoericson** [comment]
Any updates on this feature ?

**Comment by fdlsistemi** [comment]
Any updates on this feature? It is a very nice-to-have one!
